osu.Game/Screens/Menu/MenuSideFlashes.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Configuration;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Game.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Beatmaps;
+using osu.Game.Beatmaps.Timing;
+using System;
+using osu.Framework.Audio.Track;
+
+namespace osu.Game.Screens.Menu
+{
+    public class MenuSideFlashes : BeatSyncedContainer
+    {
+        public override bool HandleInput => false;
+
+        private Bindable<WorkingBeatmap> beatmap = new Bindable<WorkingBeatmap>();
+
+        private Box leftBox;
+        private Box rightBox;
+
+        private const float amplitude_dead_zone = 0.25f;
+        private const float alpha_multiplier = (1 - amplitude_dead_zone) / 0.55f;
+        private const int box_max_alpha = 200;
+        private const double box_fade_in_time = 65;
+
+        public MenuSideFlashes()
+        {
+            RelativeSizeAxes = Axes.Both;
+            Anchor = Anchor.Centre;
+            Origin = Anchor.Centre;
+            BlendingMode = BlendingMode.Additive;
+            Children = new Drawable[]
+            {
+                leftBox = new Box
+                {
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.CentreLeft,
+                    RelativeSizeAxes = Axes.Y,
+                    Width = 300,
+                    Alpha = 0,
+                    BlendingMode = BlendingMode.Additive,
+                    ColourInfo = ColourInfo.GradientHorizontal(new Color4(255, 255, 255, box_max_alpha), Color4.Transparent),
+                },
+                rightBox = new Box
+                {
+                    Anchor = Anchor.CentreRight,
+                    Origin = Anchor.CentreRight,
+                    RelativeSizeAxes = Axes.Y,
+                    Width = 300,
+                    Alpha = 0,
+                    BlendingMode = BlendingMode.Additive,
+                    ColourInfo = ColourInfo.GradientHorizontal(Color4.Transparent, new Color4(255, 255, 255, box_max_alpha)),
+                }
+            };
+        }
+
+        protected override void OnNewBeat(int newBeat, double beatLength, TimeSignatures timeSignature, bool kiai)
+        {
+            if (!beatmap?.Value?.Track?.IsRunning ?? false)

```

Comment:
I don't think this is ever going to be true, given the logic inside BeatSyncedContainer. You wouldn't want to do it this way anyway though - if you need to speed up the fade out it's better to bind to `beatmap.ValueChanged`.